.editorconfig

```diff
@@ -142,4 +142,8 @@ dotnet_diagnostic.CA2227.severity = none
 # Suppressed[CA1051] (Do not declare visible instance fields)
 # Relates to protected declarations in MongoDBTests
 # Possible solution by making private and declaring using properties
-dotnet_diagnostic.CA1051.severity = none
\ No newline at end of file
+dotnet_diagnostic.CA1051.severity = none
+
+# Suppressed[CA1054] as we expect the URI given within a request to be a string.
+# We could validate if the URI is of the correct format after receiving it.
+dotnet_diagnostic.CA1054.severity = none
```

Comment:
Why did we need to add this suppression? What was causing the CA1054 warning?

Reply:
I'm not sure but I think it's picking up variables with URI in their name and suggesting that these should not be set as strings but URI types.

Reply:
You're correct, do we want to consider using the URI type?

I'm not sure how that would work for our Infrastructure/Requqest/Response objects, how would that be serialised and deserialised.

I think for what we're doing using a string is fine and suppressing the warning is reasonable.